setup.cfg

```diff
@@ -1,5 +1,5 @@
 [metadata]
-version = 1.0.1
+version = 1.0.2
```

Comment:
It makes sense to consider this in the PR, but we actually prefer bumping the version of the repo right before we'd do a release. So if you don't mind, could you revert this back?